libs/portals/admin/form-system/src/utils/fileTypes.ts

```diff
@@ -0,0 +1,59 @@
+export const fileTypes = {
```

Comment:
This a duplication of `libs/portals/admin/form-system/src/lib/utils/fileTypes.ts`